src/app/core/utils/Reducers.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Utility reducer that can reduce a generic object with a specific "dist"
+ *  parameter. This finding an object with the minimum "distance" and then
```

Comment:
This second sentence reads weird

Reply:
Jesus I don't even know what I was trying to write there good lord